_includes/v20.1/prod-deployment/insecure-test-load-balancing.md

```diff
@@ -1,8 +1,12 @@
 CockroachDB offers a pre-built `workload` binary for Linux that includes several load generators for simulating client traffic against your cluster. This step features CockroachDB's version of the [TPC-C](http://www.tpc.org/tpcc/) workload.
 
+{{site.data.alerts.callout_info}}
+Be sure that you have [set up an inbound rule](deploy-cockroachdb-on-aws-insecure.html#step-2-configure-your-network) that allows traffic from the application to the load balancer. In this case, you will run the sample workload on one of your instances. The traffic source for your inbound rule should therefore be the **internal (private)** IP address of that instance. To find this, open the Instances section of the Amazon EC2 console and click on the instance.
```

Comment:
nit: Should "Instances" be "*Instances*" since we are referring to a UI element? If yes, need to update other places where it appears.